CDS-93927 - using secrets as URLs in an HTTP step (#6137)
harness/developer-hub: docs/continuous-delivery/x-platform-cd-features/cd-steps/utilities/http-step.md

- Method: The [HTTP method](https://restfulapi.net/http-methods/#summary) to use in the step.
- Request body: The message body of the HTTP message.

:::important

> Currently, using secret expressions as a part of URLs in HTTP steps work under the following conditions only.

Author: ```suggestion
Currently, Harness doesn't support using secret expressions as a part of URLs in HTTP steps.
```
